coda/coda_mdstore/views.py

```diff
@@ -670,21 +670,44 @@ def externalIdentifierSearch(request, identifier=None):
 def externalIdentifierSearchJSON(request):
     """External_Identifier search formatted in JSON."""
     ark = request.GET.get('ark', '')
+    showAll = request.GET.get('showAll', False)
     if ('ark:/%d' % settings.ARK_NAAN) not in ark:
         ark = 'ark:/%d/%s' % (settings.ARK_NAAN, ark)
+    data = []
+    if not showAll:
```

Comment:
Currently with this condition, it works as expected if `showAll` is not in the request at all, but if the request has parameter `showAll=False`, `showAll` is set to 'False' which is a non-empty string which evaluates to True. Thus it registers as the same as `showAll=True` and similarly is the case if `showAll` is used with any value.

Reply:
Yeah..I'll have it changed.